app/Models/Map.php

```diff
@@ -720,4 +720,69 @@ public function hasDistanceUnit(): bool
         //return false;
         return !empty($this->config['distance_measure']);
     }
+    /**
+     * Available datagrid actions
+     * @param Campaign $campaign
+     * @return string[]
+     * @throws Exception
+     */
+    public function datagridActions(Campaign $campaign): array
+    {
+        $actions = [];
+
+        // Relations & Inventory
+        if (!isset($this->hasRelations)) {
```

Comment:
This looks like copy-pasted code. You can simply call `parent::datagridActions()` to call the method from MiscModel :)